bus-mapping/src/evm/opcodes.rs

```diff
@@ -100,7 +102,7 @@ fn fn_gen_associated_ops(opcode_id: &OpcodeId) -> FnGenAssociatedOps {
         // OpcodeId::CALLER => {},
         // OpcodeId::CALLVALUE => {},
         // OpcodeId::CALLDATALOAD => {},
-        // OpcodeId::CALLDATASIZE => {},
+        OpcodeId::CALLDATASIZE => CallDataSize::gen_associated_ops,
```

Comment:
You can now use `StackOnlyOpcode` as `TIMESTAMP` opcode is already merged.

Reply:
@icemelon rebasing was a mess (because of the merge commit that got through). I'm closing this PR and opening a new one with all the changes from before as well as the changes suggested in the reviews.